sparkmagic/sparkmagic/utils/progress.py

```diff
@@ -0,0 +1,32 @@
+from ipywidgets.widgets import FloatProgress, Layout
+
+class ProgressIndicator:
+    def __init__(self, session, statement_id):
+        pass
+
+    def update(self, value):
+        pass
+
+    def close(self):
+        pass
+
+class defaultProgressIndicator(ProgressIndicator):
```

Comment:
Two minor changes:

1. Please change to `TitleCase`
```suggestion
class DefaultProgressIndicator(ProgressIndicator):
```

2. Give it a more meaningful name, one idea would be

```suggestion
class HorizontalProgressBarIndicator(ProgressIndicator):
```